lenovo/legion/16ahp9/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/gpu/nvidia
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  # Use latest LTS kernel for more Raphael fixes
+  boot = lib.mkMerge [
+    (lib.mkIf (lib.versionOlder pkgs.linux.version "6.6") {
+      kernelPackages = pkgs.linuxPackages_latest;
+      kernelParams = [ "amdgpu.sg_display=0" ];
```

Comment:
In NixOS unstable we have linux 6.12 now by default. Shouldn't be in that case this also exported?